tachyon/crypto/commitments/vector_commitment_scheme.h

```diff
@@ -69,7 +69,8 @@ class VectorCommitmentScheme {
   // size of parameters.
   template <typename ContainerTy>
   [[nodiscard]] bool Commit(const ContainerTy& container,
-                            const Field& random_value, Commitment* result) const {
+                            const Field& random_value,
+                            Commitment* result) const {
```

Comment:
Is the formatter acting weird?

Reply:
It seems so. Can you check how it's formatted in your environment?

Reply:
The issue was due to my clang-format version being 17!